docs/documentation/automation/question_exec.md

```diff
@@ -6,4 +6,105 @@ sidebar_label: Question Exec
 import useBaseUrl from '@docusaurus/useBaseUrl'; 
 import Highlight from '@theme/Highlight';
 
-# Subtitle {#subtitle}
+<span class="hero__subtitle">
+
+_Customize Survey Questions with Javascript Code_
+
+</span>
+
+## Overview
+Surveys are among Cotalker's main features. Their flexibility and customization ability allow them to adapt to each business's needs.
+
+This spec expands the notion of survey functionality by introducing _Question EXEC_, custom code execution for your survey questions.
+
+:::info
+- Custom code must be written in **Javascript**.
+- Code should always be previously _sandboxed_.
+:::
+
+## Set Up
+
+To add your custom code, choose any survey component and select the **Automation** tab. The code is added in the field at the bottom of the settings panel.
+_See the image below:_
+
+<img alt="automation tab" class="img_sizing_narrow item shadow--md" src={useBaseUrl('img/automation_question_exec_00.png')} />
+<br/>
+<br/>
+
+You can add custom code in any of the four _stages_ of a _survey component's lifecycle_. Set the **stage** either with the dropdown menu or by clicking the corresponding icon. 
+_See the image below:_
+
+<img alt="automation tab" class="img_sizing_narrow item shadow--md" src={useBaseUrl('img/automation_question_exec_01.png')} />
+<br/>
+
+## Component Lifecycle Stages
+- **Preload**: The code executes when a new survey instance is summoned.
+- **onDisplay**: Code executes when a survey is in editing mode.
```

Comment:
Each time the survey is shown, in editing mode

Reply:
So "editing mode" is whenever a survey pops-up and is ready to be answered and sent?

Reply:
So "editing mode" is whenever a survey pops up and is ready to be answered and sent?